### PR TITLE
Update PO box compliance checker version

### PR DESCRIPTION
**Pull request by anguss00**
@mhidas @ggalibert @lbesnard this updates the version on a fresh po box, you potentially need to uninstall the old plugin and compliance checker if updating an existing box.

### CODE REVIEW

**Comment by lwgordonimos** [comment]
@anguss00 Actually this whole node can be removed, since it's prebuilt in the EGG, and nothing uses that git info any more:

https://github.com/aodn/chef/commit/7736a03db19196f0b94a1c8a246cf99c3b631768

**Comment by anguss00** [comment]
Only thing is, the node allows the base checker branch to be set. The default set in attributes is IOOS main, while we have the aodn fork overwritten. I agree though, it's ripe for just configuring on the attribute

**Comment by lwgordonimos** [comment]
See https://github.com/aodn/data-services/pull/523#issuecomment-231636662

File: cookbooks/imos_po/attributes/netcdf_checker.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 default['imos_po']['netcdf_checker']['dir']           = "/usr/local/bin"
-default['imos_po']['netcdf_checker']['repo']          = "https://github.com/ioos/compliance-checker.git"
 default['imos_po']['netcdf_checker']['branch']        = "master"
```
Comment (lwgordonimos) [inline cookbooks/imos_po/attributes/netcdf_checker.rb]: This ['repo'] line can also be removed as it's now unused in the client recipe. The ['dir'] is only used by the manual symlink, but I don't think it's worth doing any work to remove that given the future plans for this.

Comment (anguss00) [inline cookbooks/imos_po/attributes/netcdf_checker.rb]: Ah yes, should definitely remove the "branch" line. Will amend

**Comment by anguss00** [comment]
Doneskis @lwgordonimos

**Pull request merged**